projects/components/src/beta-tag/beta-tag.component.test.ts

```diff
@@ -1,16 +1,21 @@
 import { createHostFactory, Spectator } from '@ngneat/spectator/jest';
+import { LabelTagComponent } from '../label-tag/label-tag.component';
 import { BetaTagComponent } from './beta-tag.component';
 
 describe('Beta Tag Component', () => {
   let spectator: Spectator<BetaTagComponent>;
 
   const createHost = createHostFactory({
     component: BetaTagComponent,
+    entryComponents: [LabelTagComponent],
```

Comment:
shouldn't need an entry component here - maybe a declaration if we want to go one level deep on the test (current approach), or to keep it strictly shallow, a mocked label tag component.

Reply:
tried doing that. the Color enum within LabelTag component doesn't get resolved when I use a declaration. Any alternate ideas? @aaron-steinfeld

Reply:
That's strange - I'd suspect that would be an artifact of test caching, as an entry vs declaration shouldn't be relevant there. They're sometimes a bit squirrelly around the enums I've found. If it's still happening after switching back to declaration now, try clearing the jest cache (`--clear-cache` arg on whatever test command you're running) and rerunning - and if that doesn't work, we can leave it as an entry component.

Reply:
it was the jest cache. Works after clearing. Updated.